interfaces/builtin/mount_control.go

```diff
@@ -296,7 +296,7 @@ func validateWhatAttr(mountInfo *MountInfo) error {
 		return fmt.Errorf(`mount-control "what" setting cannot be used: %v`, err)
 	}
 
-	// "what" must be set to "none" iff the type is "tmpfs"
+	// "what" must be set to "none" if the type is "tmpfs"
```

Comment:
I don't think this was a typo, usually `iff` denotes `if and only if`

Source: https://www.merriam-webster.com/dictionary/iff